.github/ISSUE_TEMPLATE/user_bug_report.yml

```diff
@@ -0,0 +1,89 @@
+name: User bug report
+description: Create a report to help us improve
+title: '[User reported bug]: '
+labels: C-bug, S-needs-triage
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      value: "I expected to see this happen:
+
+
+      Instead, this happened:
+
+    "
+    validations:
+      required: true
+  - type: textarea
+    id: reproduce
+    attributes:
+      label: Steps to reproduce
+      description: Copy and paste the exact commands or code here.
+      placeholder: "Behavior or code sample that causes the bug"
+    validations:
+      required: false
+  - type: textarea
+    id: logs
+    attributes:
+      label: Zebra logs
+      description: Copy and paste the last 100 Zebra log lines or upload the full logs to https://gist.github.com/ and add a link to them here.
+      placeholder: "copy and paste the logs here"
+    validations:
+      required: false
+  - type: checkboxes
+    id: crate
+    attributes:
+      label: In what crates the bug is located, if in any?
+      description: You may select more than one.
+      options:
+        - label: zebrad
+        - label: zebra-chain
+        - label: zebra-consensus
+        - label: zebra-network
+        - label: zebra-node-services
+        - label: zebra-rpc
+        - label: zebra-script
+        - label: zebra-state
+        - label: zebra-test
+        - label: zebra-utils
+        - label: tower-batch
+        - label: tower-fallback
+  - type: input
+    id: zebrad-version
+    attributes:
+      label: Zebra Version
+      description: "For bugs in `zebrad`, run `zebrad version`."
+    validations:
+      required: false
+  - type: textarea
+    id: libraries
+    attributes:
+      label: Zebra libraries
+      description: For bugs in the `zebra` libraries, list the `zebra` crate versions.
+      placeholder: "You can get the crate version by checking the Cargo.toml inside the crate folder."
+    validations:
+      required: false
+  - type: checkboxes
+    id: os
+    attributes:
+      label: Which operating systems have you used?
+      description: You may select more than one.
+      options:
+        - label: macOS
+        - label: Windows
+        - label: Linux
+  - type: input
+    id: os-details
+    attributes:
+      label: OS details
+      description: "Linux, macOS, BSD: the output of `uname -a`
+Windows: Windows version and 32-bit or 64-bit"
+      placeholder:
+    validations:
+      required: false
```

Comment:
Please add "Is there anything else that could help us solve this issue?" as a free text field. Otherwise people will try to answer that question in other fields, which is confusing.